### PR TITLE
[12.0][IMP] connector_jira: import estimation and status for tasks

### PR DESCRIPTION
**Pull request by alexey-pelykh**
- [x] Status
- [x] Estimated hours

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@guewen @simahawk please review

**Review by artemsenko — APPROVED** [review]
Functional test - LGTM

File: connector_jira/models/project_task/importer.py [hunk 1]
```diff
@@ -108,6 +111,35 @@ def parent(self, record):
...
+        )
+        return {'stage_id': stage_id.id if stage_id else False}
+
+    @mapping
+    def time_estimate(self, record):
+        original_estimate = record['fields'].get('timeoriginalestimate')
+        if not original_estimate:
+            return {'planned_hours': False}
+        planned_hours = (
+            timedelta(seconds=int(original_estimate)) / timedelta(hours=1)
```
Comment (guewen) [inline connector_jira/models/project_task/importer.py]: Is there a reason for needing `timedelta` here? `float(original_estimate) / 3600.` is equal.

**Review by guewen — APPROVED** [review]

File: connector_jira/models/project_task/importer.py [hunk 2]
```diff
@@ -108,6 +111,35 @@ def parent(self, record):
...
+    def status(self, record):
+        status = record['fields'].get('status', {})
+        status_name = status.get('name')
+        if not status_name:
+            return {'stage_id': False}
+        project_binder = self.binder_for('jira.project.project')
+        project_id = project_binder.unwrap_binding(
+            self.options.project_binding
+        )
+        stage_id = self.env['project.task.type'].search(
```
Comment (simahawk) [inline connector_jira/models/project_task/importer.py]: you get a stage record here, not an id. Also, stage.id will give you already False if stage is not found: no need for the IF below ;)

**Review by simahawk — APPROVED** [review]

File: connector_jira/models/project_task/importer.py (hunk 2 above)
Comment (alexey-pelykh) [reply connector_jira/models/project_task/importer.py]: As you wish :)) Changed

File: connector_jira/models/project_task/importer.py (hunk 1 above)
Comment (alexey-pelykh) [reply connector_jira/models/project_task/importer.py]: No specific reason at all, just other way of expressing the same

**Comment by alexey-pelykh** [comment]
@guewen comment attended, thanks

**Comment by alexey-pelykh** [comment]
@guewen I guess we're good to merge?

**Pull request merged**